src/transformer/__tests__/__helpers__/highlights.ts

```diff
@@ -289,6 +290,79 @@ export const createTestModelMapWithKeywords = () => {
 
   return modelMap
 }
+export const createTestModelMapWithKeywordsAndAuthorQuery = () => {
+  const modelMap = new Map<string, Model>()
+
+  const project: Project = {
+    objectType: ObjectTypes.Project,
+    _id: 'MPProject:1',
+    createdAt: 0,
+    updatedAt: 0,
+    sessionID: 'test',
+    owners: [],
+    writers: [],
+    viewers: [],
+  }
+
+  const manuscript: Manuscript = {
+    objectType: ObjectTypes.Manuscript,
+    _id: 'MPManuscript:1',
+    createdAt: 0,
+    updatedAt: 0,
+    containerID: project._id,
+    sessionID: 'test',
+  }
+
+  const keyword: Keyword = {
+    objectType: ObjectTypes.Keyword,
+    _id: 'MPKeyword:1',
+    createdAt: 0,
+    updatedAt: 0,
+    containerID: project._id,
+    sessionID: 'test',
+    containedGroup: 'MPKeywordGroup:test',
+    name: 'test',
+  }
+
+  modelMap.set(keyword._id, keyword)
+
+  manuscript.keywordIDs = [keyword._id]
+
+  modelMap.set(manuscript._id, manuscript)
+
+  const comment: CommentAnnotation = {
+    _id: 'MPCommentAnnotation:3AAEE869-7DC9-4392-B8C6-9C5FCA522120',
+    objectType: 'MPCommentAnnotation',
+    target: 'MPKeywordGroup:test',
```

Comment:
this should be section ID.
if you looked at the keyword it will be like this:
```
{
            "type": "keywords_section",
            "attrs": {
                "id": "MPSection:1EA32707-96E5-4CFF-856E-5AD75D814295",
                "dataTracked": null
            },
```